pkg/helm/uninstall_test.go

```diff
@@ -65,8 +64,7 @@ func TestMixin_Uninstall(t *testing.T) {
 	defer os.Unsetenv(test.ExpectedCommandEnv)
 	for _, uninstallTest := range uninstallTests {
 		t.Run(uninstallTest.expectedCommand, func(t *testing.T) {
-			os.Setenv(test.ExpectedCommandEnv,
-				strings.Join([]string{"helm init --upgrade --wait", uninstallTest.expectedCommand}, "\n"))
+			os.Setenv(test.ExpectedCommandEnv, uninstallTest.expectedCommand)
```

Comment:
Good question; indeed, b/c the default mock returns the same Tiller version as the default client we install on build.  Since this is the first check in `Init` and it matches, no further init steps need be taken (Tiller is running and version matches; we're good to go!)

Would a comment in this install/uninstall/etc. tests be helpful?  Or perhaps another explicit 'happy path' test in `init_test.go` showing this?

Reply:
Nah I just wasn't sure why we had that line added to the test and then removed it in the next commit. I was reviewing commit by commit and wasn't sure why it changed.